privs.lua

```diff
@@ -1,11 +1,17 @@
 local S = minetest.get_translator("travelnet")
 
-minetest.register_privilege("travelnet_attach", {
-	description = S("allows to attach travelnet boxes to travelnets of other players"),
-	give_to_singleplayer = false
-})
+minetest.register_on_mods_loaded(function()
+	if not minetest.registered_privileges[travelnet.attach_priv] then
+		minetest.register_privilege(travelnet.attach_priv, {
+			description = S("allows to attach travelnet boxes to travelnets of other players"),
+			give_to_singleplayer = false
+		})
+	end
```

Comment:
I think this registration probably should only happen with default privilege but never with custom privilege (or should be explicit).
Reason for this is that it complicates things as you can see it requires deferred execution to get around conflicts, it also allows typos to get through without any warnings.

I do believe it should fail with unregistered custom privileges as I cannot really imagine any situation where both unregistered custom privilege is needed and default privilege is not enough. Do you know situation where it would be needed?